src/handlers/miner_hbbft_handler.erl

```diff
@@ -229,6 +231,14 @@ handle_command({txn, Txn}, State=#state{hbbft=HBBFT}) ->
                     {reply, ok, fixup_msgs(Msgs), State#state{hbbft=NewHBBFT}}
             end
     end;
+handle_command(maybe_skip, State = #state{hbbft = HBBFT,
```

Comment:
I would argue that this should replace the old skip command entirely, as the old one is no longer safe and very easily leads to misalignment

Reply:
done